src/config.js

```diff
@@ -194,6 +194,43 @@ const config = {
         isPercent: true
       }]
     }
+  },
+  charts: {
+    histograms: {
+      people: {
+        field: 'den14',
+        title: 'HISTOGRAM_PEOPLE_TITLE'
+      },
+      households: {
+        field: 'hh14',
+        title: 'HISTOGRAM_HOUSEHOLDS_TITLE'
+      },
+      economy: {
+        field: 'ent14',
+        title: 'HISTOGRAM_ECONOMY_TITLE'
+      },
+      infrastructure: {
+        field: 'pcwat10',
+        title: 'HISTOGRAM_INFRASTRUCTURE_TITLE'
+      }
+    },
+    height: 200,
+    width: undefined, // Width is dynamically detected from page state
+    callouts: { // Callouts are the vertical bars representing specific values
+      country: {
```

Comment:
It's all internal references, these match the keys used in the `SoumData.compare` output and allow us to override the soum label in code (Which shouldn't have a label here because it's never used...). I'll put in a comment explaining the key's meaning.